NOTICE.md

```diff
@@ -29,6 +29,11 @@ The project maintains the following source code repositories:
 
 * https://github.com/eclipse/packages
 
+# Copyright holders
+
+* Copyright 2019 Bosch Software Innovations GmbH
```

Comment:
As we are adding this section, I would like to see "Red Hat Inc" there as well :wink:

Reply:
done ..